### PR TITLE
V4.0.2 Spanish translation (based in October 2020, EN release)

### PR DESCRIPTION
**Pull request by CarlosAllendes**
V4.0.2 Spanish translation (based in October 2020, EN release)

### CODE REVIEW

**Comment by CarlosAllendes** [comment]
Hi @tghosth
we are OK, task completed !!

File: 4.0/es/0x01-Frontispiece.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Frontispiece
+
+## Acerca del Estándar
+
+La norma de verificación de la seguridad de las aplicaciones (ASVS) es una lista de requisitos o pruebas de seguridad de las aplicaciones que pueden utilizar los arquitectos, desarrolladores, probadores, profesionales de la seguridad, proveedores de herramientas y consumidores para definir, construir, probar y verificar aplicaciones seguras.
+
+## Copyright y Licencia
+
+Version 4.x (Bleeding Edge version), 2021
```
Comment (tghosth) [inline 4.0/es/0x01-Frontispiece.md:9]: Please can you confirm you definitely based off the correct version as I can see this line is different to what is in the 4.0.2 version: https://github.com/OWASP/ASVS/blob/v4.0.2/4.0/en/0x01-Frontispiece.md

File: 4.0/es/0x00-Header.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#
+
+![LOGO OWASP](../images/owasp_logo_1c_notext.png)
+
+# Application Security Verification Standard 4.0
```
Comment (tghosth) [inline 4.0/es/0x00-Header.md:5]: Should this be 4.0.2?

File: 4.0/es/0x01-Frontispiece.md (hunk 1 above)
Comment (CarlosAllendes) [reply 4.0/es/0x01-Frontispiece.md:9]: Dear Josh ...
Sorry, we made a mistake.

We, the team, could not properly copy and paste from our 100% Spanish translated PDF file. And then rewrite them as markdown files. 
Just for a coincidence. Right now we are completing the execution of a CybersecTour in several Latin countries ... And we are very close to finishing the talks.
As soon as we release collaborators, we will ask them to do a new, more intense and personal QA.

Best Regards.

**Comment by jmanico** [comment]
Can you take a look at this when you have a chance @tghosth ?

**Comment by tghosth** [comment]
@jmanico I think they are still working on QA

**Comment by tghosth** [comment]
Hi all, any progress on this translation?

**Comment by hansphp** [comment]
@CarlosAllendes & @tghosth, I would like to contribute to the translation in Spanish, how can I help?

**Comment by CarlosAllendes** [comment]
> @CarlosAllendes & @tghosth, I would like to contribute to the translation in Spanish, how can I help?

Hi Hans,  currently the document ASVS is  100% translated to spanish. BUT WE have some problems to merge our modifications, because the person that offer support in that task is not longer available.  CAN YOU help us  ?

**Comment by hansphp** [comment]
Hello!,
@CarlosAllendes, I understand, of course.
In fact I just forked your repository "CarlosAllendes/ASVS" and I am doing a reverification of the contents to align them to the context of the stable version 4.0.2. to later request to merge both branches.

For example:
image

If you have something additional and it is offline we can merge it to make the best possible translation.

Greetings

**Review by jmanico — APPROVED** [review]

**Comment by jmanico** [comment]
We just need 2 more reviews. cc @vanderaj @elarlang @danielcuthbert @tghosth

**Review by jmanico — APPROVED** [review]

**Comment by hansphp** [comment]
We have continued work on the issue #1129 and pr #1130

**Comment by jmanico** [comment]
Since we have a new 4.0.3 PR I'm going to close this

**Pull request closed without merging**